packages/desktop-client/src/components/reports/spreadsheets/filterEmptyRows.ts

```diff
@@ -0,0 +1,19 @@
+// @ts-strict-ignore
+import { type GroupedEntity } from 'loot-core/types/models/reports';
+
+export function filterEmptyRows(
+  showEmpty: boolean,
+  data: GroupedEntity,
+  balanceTypeOp: string,
+): boolean {
+  let showHide;
+  if (balanceTypeOp === 'totalTotals') {
+    showHide =
+      data['totalDebts'] !== 0 ||
+      data['totalAssets'] !== 0 ||
+      data['totalTotals'] !== 0;
+  } else {
+    showHide = data[balanceTypeOp] !== 0;
```

Comment:
Having trouble figuring out this error `because expression of type 'string' can't be used to index type 'GroupedEntity'`